test/mocks/MockHTTP.swift

```diff
@@ -50,9 +50,9 @@ extension MockHTTP {
     
 extension MockHTTP {
     typealias RequestValidator = (URLRequest) -> ()
-    
-    static var responses: [(statusCode: Int, data: Data?, headers: [String: String]?, error: Error?, validate: RequestValidator?)] = []
-    
+
+    static nonisolated(unsafe) var responses: [(statusCode: Int, data: Data?, headers: [String: String]?, error: Error?, validate: RequestValidator?)] = []
```

Comment:
such a peculiar language, which requires you to define what functionaccess

Reply:
Since it's a global variable that's not a constant (i.e., `var` and not `let`) there's no real way to make read/writes to it thread-safe, so marking it as `nonisolated(unsafe)` is basically telling the compiler "I know what I'm doing, don't fail compilation on this".